src/strings/PowerFxResources.en-US.resx

```diff
@@ -5286,4 +5286,8 @@
   <data name="AboutPlainText_text" xml:space="preserve">
     <value>The text to have its HTML and XML tags removed.</value>
   </data>
+  <data name="ErrFilterFunctionBahaviorAsPredicate" xml:space="preserve">
+    <value>Filter functions don't allow behavior function as predicate.</value>
```

Comment:
>don't allow behavior function [](http://example.com/codeflow?start=28&length=29)

Suggestion: "...don't allow behavior functions as predicates" (plural in 'function' and 'predicate')